src/tsolvers/liasolver/Matrix.cpp

```diff
@@ -0,0 +1,1053 @@
+//
+// Created by prova on 24.08.18.
+//
+
+#include <liasolver/Matrix.h>
+
+LAVecRef
+LAVecStore::getNewVec(const vec<opensmt::Real>& ps, const opensmt::Real& den)
+{
+    LAVecRef vr = lva.alloc(ps, den);
+    return vr;
+}
+
+char*
+LAVecStore::print(LAVecRef vr)
+{
+    char *str = (char*) malloc(1);
+    str[0] = '\0';
+    asprintf(&str, "[\n");
```

Comment:
This creates a warning in gcc: "unused result"
There are a couple of others, can you fixed this?

Reply:
done.

Reply:
This warning is still there, see the build: https://travis-ci.com/usi-verification-and-security/opensmt/jobs/262277924

It's complaining about the unused return value from asprintf method.